docs/http.md

```diff
@@ -69,7 +69,7 @@ Where the elements follow the following format:
 
 | Element    | Values                                                 |
 |------------|--------------------------------------------------------|
-|bearing     |`{value},{range}` `integer 0 .. 360,integer 0 .. 180`  |
+|bearing     |`{value},{range}` `integer 0 .. 360,integer 1 .. 180`  |
```

Comment:
This is an API change. Are bearing range-0 values always impossible queries? If so, why don't we simply return directly and keep the API valid for now?